src/Features/CSharp/Portable/Completion/KeywordRecommenders/ThrowKeywordRecommender.cs

```diff
@@ -40,7 +38,7 @@ protected override bool IsValidContext(int position, CSharpSyntaxContext context
             if (context.TargetToken.Kind() is SyntaxKind.QuestionToken or
                 SyntaxKind.ColonToken)
             {
-                return context.TargetToken.Parent.Kind() == SyntaxKind.ConditionalExpression;
+                return context.TargetToken.Parent.IsKind(SyntaxKind.ConditionalExpression);
```

Comment:
undo plz.

Reply:
ah, i'm ok with this.

Reply:
Reasoning? As per discussion with @Youssef1313 above, this is completely safe. IDE doesn't complain in any way either